packages/gcp-cloud-run/src/executors/deploy/deploy.impl.ts

```diff
@@ -118,6 +118,7 @@ export async function deployExecutor(
     return { success: false }
   }
 
+  // @TODO(mkdior) - cd deprecated, should we remove?
```

Comment:
As for this, what do you guys think? Check [this](https://cloud.google.com/stackdriver/docs/deprecations/debugger-deprecation) out.

Reply:
```txt
Cloud Debugger was deprecated on May 16, 2022 and the service was shut down on May 31, 2023. You can continue to use the open source Snapshot Debugger. Snapshot Debugger was archived on September 7, 2023, so it is not receiving bug fixes or security patches. Snapshot Debugger remains available for use. You can also fork the repository and maintain your own version.
```

Reply:
Yea good one, let's remove it.